frontend/src/app/questions/questions.component.html

```diff
@@ -24,56 +24,56 @@
                     [disabled]="!selectedQuestions || !selectedQuestions.length" />
             </div>
         </p-toolbar>
-        <p-table
-            sortField="id"
-            [sortOrder]="1"
-            [columns]="cols"
-            [value]="questions"
-            [(selection)]="selectedQuestions"
-            datakey="id"
-            [tableStyle]="{ 'min-width': '50rem' }"
-            [paginator]="true"
-            [rows]="5"
-            [rowsPerPageOptions]="[5, 10, 20]"
-            styleClass="p-datatable-gridlines-striped">
-            <ng-template pTemplate="caption">
-                <div class="flex">
-                    <h3 class="m-0">Manage Questions</h3>
-                </div>
-            </ng-template>
-            <ng-template pTemplate="header" let-columns>
-                <tr>
-                    <th style="width: 10%"><p-tableHeaderCheckbox /></th>
-                    <th pSortableColumn="id" style="width: 10%">Id <p-sortIcon field="id" /></th>
-                    <th style="width: 10%">Title</th>
-                    <th style="width: 40%">Description</th>
-                    <th style="width: 10%">Topics</th>
-                    <th style="width: 10%">Difficulty</th>
-                    <th style="width: 10%"></th>
-                </tr>
-            </ng-template>
-            <ng-template pTemplate="body" let-question>
-                <tr>
-                    <td>
-                        <p-tableCheckbox [value]="question" />
-                    </td>
-                    <td>{{ question.id }}</td>
-                    <td>{{ question.title }}</td>
-                    <td style="white-space: pre-wrap">{{ question.description }}</td>
-                    <td>{{ question.topics.join(', ') }}</td>
-                    <td>{{ question.difficulty }}</td>
-                    <td>
-                        <p-button
-                            label="Edit"
-                            severity="primary"
-                            icon="pi pi-file-edit"
-                            class="mr-2"
-                            [text]="true"
-                            (onClick)="editQuestion(question)" />
-                    </td>
-                </tr>
-            </ng-template>
-        </p-table>
+        <div class="table-container" style="max-width: 1000px; margin: 0 auto">
```

Comment:
the div was to set a max width to how big the table can get. When it gets too "fat" it looks ugly. I made this not considering at how it would look with admin status, hence only affecting p-table. Now, with additional buttons, admin is at 1150px so that the table does not need to scroll and normal user is at 900px